<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="model_file" default="'$(find beacons_gazebo)/models/beacon/model.sdf'" />
    <arg name="height" default="15" />

    <node name="model_spawn__beacon_1" pkg="gazebo_ros" type="spawn_model" output="screen" args="-x -0.389 -y 8.328 -z 11.19 -R 0 -P 0 -Y 0 -sdf -model beacon_1 -file $(arg model_file)" />
    <node name="model_spawn__beacon_2" pkg="gazebo_ros" type="spawn_model" output="screen" args="-x 28.25 -y -12.749 -z 5.134 -R 0 -P 0 -Y -2.942 -sdf -model beacon_2 -file $(arg model_file)" />
    <node name="model_spawn__beacon_3" pkg="gazebo_ros" type="spawn_model" output="screen" args="-x -25.14 -y -8.81 -z 11.23 -R 0 -P 0 -Y 2.484 -sdf -model beacon_3 -file $(arg model_file)" />
    <node name="model_spawn__beacon_4" pkg="gazebo_ros" type="spawn_model" output="screen" args="-x 63.38 -y -13.88 -z 3.989 -R 0 -P 0 -Y -2.7   -sdf -model beacon_4 -file $(arg model_file)" />

</launch>